### DISCUSSION TITLE
time slider using tiff images

### DISCUSSION THREAD

**Discussion by oraibalmegdadi** [opengeos/opengeos.github.io#4] [Q&A] [answered]
I am new to using geemap, I am trying to create a time slider to show a group of TIFF files I created using Google Earth Engine. I am using colab and following this tutorial: https://leafmap.org/notebooks/79_timeseries/

```
drive_folder_path ='https://drive.google.com/file/d/1hYs1abaPl2La58CagSh9pbaxirc_kcU4/view?usp=sharing'
leafmap.download_file(drive_folder_path)
# Create a Map instance
m = geemap.Map()
# Add time slider with Google Drive folder URL
m.add_time_slider( 'geemap',
                  time_interval=0.5, 
                  position='bottomright', 
                  band=[1, 2, 3], 
                  zoom_to_layer=True)

# Display the map
m

```
The code gives me this error: TypeError: The ee_object must be an ee.Image or ee.ImageCollection

The link already contains a folder named geemap and contains a TIFF file with names DDMMYYYY

**Comment by giswqs** [comment, marked as answer]
For visualizing local files, use leafmap: https://leafmap.org/notebooks/79_timeseries

For visualizing Earth Engine images, use geemap: https://geemap.org/notebooks/62_time_slider
